### PR TITLE
BG-10340: do not sign tbsv taint inputs

### PR DESCRIPTION
**Pull request by paddyquinn**

### CODE REVIEW

**Comment by tylerlevine** [comment]
What's the context for this change? I thought we already had this logic?

**Comment by paddyquinn** [comment]
We have this logic for BCH. We do not have it for BSV. However, since BSV inherits from BCH _and_ they use the same provider address we do not need it for BSV. However, TBSV obviously cannot use the prod provider address, hence the change.

File: src/v2/coins/bsv.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ class Bsv extends Bch {
     return 'Bitcoin SV';
   }
 
+  getAddressPrefix() {
+    return 'bitcoinsv';
```
Comment (argjv) [inline src/v2/coins/bsv.js]: I don't think they changed the bech32 address prefix, and if they did, this is may be a problem when reading old BCH addresses.

File: src/v2/coins/tbsv.js [hunk 2]
```diff
@@ -15,7 +15,16 @@ class Tbsv extends Bsv {
   }
 
   getAddressPrefix() {
-    return 'bchtest';
+    return 'bsvtest';
```
Comment (argjv) [inline src/v2/coins/tbsv.js]: Ditto

File: src/v2/coins/bsv.js (hunk 1 above)
Comment (paddyquinn) [reply src/v2/coins/bsv.js]: Removed.

File: src/v2/coins/tbsv.js (hunk 2 above)
Comment (paddyquinn) [reply src/v2/coins/tbsv.js]: Removed.

**Pull request merged**